src/libraries/Microsoft.PowerFx.Core/Public/ExpressionError.cs

```diff
@@ -170,4 +170,18 @@ internal static IEnumerable<ExpressionError> New(IEnumerable<IDocumentError> err
             }
         }
     }
+
+    internal class ExpressionErrorComparer : EqualityComparer<ExpressionError>
+    {
+        // We compare only Message
```

Comment:
>// We compare only Message [](http://example.com/codeflow?start=8&length=26)

What would happen if there are 2 same messages but at different spans?

Reply:
The messages are considered different since they happened at different locations.